### DISCUSSION TITLE
Need to optimize a News Trade EA

### DISCUSSION THREAD

**Discussion by Peji** [EA31337/EA31337-classes#690] [Q&A]
I need advice regarding a News Trade EA, I want to trade at specific news, the EA needs to over come and first few seconds of zigzag and fake moves in the market. Please advise?

**Comment by kenorb** [comment]
Either you import all news into pre-defined list or use the Web class to connect external endpoints. To do that, you need to add specific hosts to the whitelist in the platform. For ZigZag signals you can use Indi_ZigZag class.
